specs/csr/csr.yml

```diff
@@ -12,3 +12,4 @@ spec:
   - key encipherment
   - server auth
   - client auth
+  signerName: kubernetes.io/kube-apiserver-client-kubelet
```

Comment:
@xiujuanx  You seems not change `system:masters`, could you please help explain this change?

Reply:
This csr case was disabled, xiujuan is working on it.

Reply:
OK